### ISSUE TITLE
accessibility  

### ISSUE THREAD

**Issue by noorafi** [FACN4/week5_The_Badboyz#28]
im sorry but this point the accessibility should be atleast higher that 60.
![screenshot from 2018-08-02 10-37-22](https://user-images.githubusercontent.com/33840142/43569568-880529c4-9640-11e8-8b66-be5009cfc041.png) 
:face_with_head_bandage:
